data-prepper-plugins/geoip-prepper/build.gradle

```diff
@@ -0,0 +1,41 @@
+plugins {
+    id 'java'
+}
+
+dependencies {
+    implementation project(':data-prepper-api')
+    implementation project(':data-prepper-plugins:common')
+    implementation 'commons-codec:commons-codec:1.15'
+    testImplementation project(':data-prepper-api').sourceSets.test.output
```

Comment:
nit - keep the `implementations` as a group at the top, then then the `testImplementations` after